java.d/src/main/java/org/firehol/netdata/utils/NetdataLevel.java

```diff
@@ -0,0 +1,40 @@
+package org.firehol.netdata.utils;
+
+import java.util.logging.Level;
+
+/**
+ * Log levels defined by Netdata for {@code error.log} (plugin's stderr).
+ * 
+ * @see <a href="https://github.com/firehol/netdata/wiki/Log-Files#errorlog">Log-Files in Netdata wiki</a>
+ */
+public class NetdataLevel extends Level {
```

Comment:
Netdata Level does not tell us anything about logging. Why not `LogLevel`?

Reply:
I am happy with the name `NetdataLevel`, here ius why: it is meant to express a `Level` as defined by Netdata, while `LogLevel` would not clarify how it differs from the base class `Level` (which is already meant for logging, hence the package `java.util.logging`). 
Other subclasses of `Level` are named similarly, e.g.:
- `org.jboss.logging.JDKLevel`
- `oracle.jdbc.driver.OracleLog$OracleLevel`

To further emphasize its role in logging, should we rather move it into a package with `logging` in its name or have it as a nested class of `LoggingUtils`?

Reply:
Ok. You convinced me. This are good arguments. I like it in that package. I think nested classes should have a logical reason. I do not see one here.